depsolver.js

```diff
@@ -6,6 +7,7 @@ const fs = require('fs');
 function resolvePath(file, x) {
   if (x.startsWith('@rpath')) {
     let baseIndex = file.lastIndexOf('/Frameworks');
+    let basePath = '/';
```

Comment:
Obvervation: The initial value of `basePath` is _only_ kept when `file` contains neither `/Frameworks` nor `/`